files/image_config/corefile_uploader/core_uploader.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Host core file uploader daemon
+Requires=updategraph.service
+After=updategraph.service
```

Comment:
why does this service depends on updategraph service? can this be removed?

Reply:
Agree. Switched to syslog.service